skrf/io/touchstone.py

```diff
@@ -507,18 +507,18 @@ def get_noise_data(self):
 
     def is_from_hfss(self):
         """
-        Check if the Touchstone file has been produced by HFSS.
+        Check if the Touchstone might have complex port impedances for each frequency.
```

Comment:
Exporting the characteristic impedances (which might be complex valued) is an option in HFSS, which can be unticked. ,

Reply:
I changed it since this function is also used to check for port impedances in files exported by skrf. The name doesn't really match anymore though.